website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -201,245 +267,293 @@ UpstreamConfig = {
       type: 'UpstreamConfiguration: <optional>',
       description: `Controls default configuration settings that apply across all upstreams, and per-upstream
                     configuration overrides. Note that per-upstream configuration applies across all federated datacenters
-                    to the pairing of source and upstream destination services. 
+                    to the pairing of source and upstream destination services.
                     Added in v1.10.0.`,
       children: [
```

Comment:
these docs I added some alternate descriptions for yaml to reference the yaml keys instead of the hcl keys, e.g. `connectMilliseconds` vs `ConnectMilliseconds`